docs/emr/presto-admin-installation-on-emr.rst

```diff
@@ -0,0 +1,32 @@
+=========================
+Presto Admin Installation 
```

Comment:
There's probably a better way to phrase it or structure it, but to me it seems that this really isn't something specific to emr.  It's a problem lots of people could have, so we should instead have better documentation in the main documentation.